ui/src/components/__TEST__/util.js

```diff
@@ -58,15 +62,17 @@ const AllTheProviders = ({ children }) => {
     },
   };
   return (
-    <IntlProvider locale="en" messages={translations_en}>
-      <Provider store={store}>
-        <QueryClientProvider client={queryClient}>
-          <AlertProvider>
-            <ThemeProvider theme={theme}>{children}</ThemeProvider>
-          </AlertProvider>
-        </QueryClientProvider>
-      </Provider>
-    </IntlProvider>
+    <Router history={history}>
```

Comment:
can you please explain to me why we need this Wrapper?

Reply:
@ChengYanJin , because we use react-router-dom's `Link` in the component, without the Router wrapper this makes the test suite fail because react-router component cannot be rendered outside a Router context.

Here we create an in-memory fake history to be provided to the test suite.